.github/workflows/greetings.yml

```diff
@@ -26,7 +26,7 @@ jobs:
     name: Greeting
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/first-interaction@v1
+      - uses: actions/first-interaction@v1.1.1
         with:
```

Comment:
This modification shouldn't be part of this PR. The good thing is that this update won't affect the current workflow configuration. Acceptable.